### PR TITLE
*/*: fix various broken M4 libdir macros

### PR DESCRIPTION
**Pull request by thesamesam**

### CODE REVIEW

File: media-libs/opus/opus-1.3.1-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.3.1-libdir-macro.patch
+)
+
+multilib_src_configure() {
+	local myeconfargs=(
+		$(use_enable custom-modes)
+		$(use_enable doc)
+		$(use_enable static-libs static)
```
Comment (SoapGentoo) [inline media-libs/opus/opus-1.3.1-r2.ebuild:31]: sure?

File: media-libs/opus/opus-1.3.1-r2.ebuild [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	)
+
+	for i in ${INTRINSIC_FLAGS} ; do
```
Comment (SoapGentoo) [inline media-libs/opus/opus-1.3.1-r2.ebuild:34]: declare `local i`

File: media-libs/opus/opus-1.3.1-r2.ebuild [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+		use ${i} && myeconfargs+=( --enable-intrinsics )
+	done
+
+	if is-flagq -ffast-math || is-flagq -Ofast; then
+		myeconfargs+=( "--enable-float-approx" )
```
Comment (SoapGentoo) [inline media-libs/opus/opus-1.3.1-r2.ebuild:39]: remove quotes

File: media-libs/libogg/libogg-1.3.4-r1.ebuild [hunk 4]
```diff
@@ -15,6 +16,10 @@ IUSE="static-libs"
 
 DOCS=( AUTHORS CHANGES )
```
Comment (SoapGentoo) [inline media-libs/libogg/libogg-1.3.4-r1.ebuild:17]: required?

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

**Review by dm0- — COMMENTED** [review]
Where does this fix the intrinsics config bug?  This just adds whitespace to `-r1`, and the only difference between `-r1` and `-r2` is adding a patch.

File: media-libs/libogg/libogg-1.3.4-r1.ebuild (hunk 4 above)
Comment (thesamesam) [reply media-libs/libogg/libogg-1.3.4-r1.ebuild:17]: No!

File: media-libs/opus/opus-1.3.1-r2.ebuild (hunk 2 above)
Comment (thesamesam) [reply media-libs/opus/opus-1.3.1-r2.ebuild:34]: ofc

File: media-libs/opus/opus-1.3.1-r2.ebuild (hunk 1 above)
Comment (thesamesam) [reply media-libs/opus/opus-1.3.1-r2.ebuild:31]: static-libs or whether or not I'd actually fixed the bug? I've fixed the latter now, but we can see if removing it is possible.

**Comment by thesamesam** [comment]
@dm0 That's called "losing stuff in fixing conflicts and rebasing". Fixed now.

File: media-libs/libogg/libogg-1.3.4-r1.ebuild [hunk 5]
```diff
@@ -24,6 +27,6 @@ multilib_src_configure() {
 }
 
 multilib_src_install_all() {
-	einstalldocs
+	default
```
Comment (SoapGentoo) [inline media-libs/libogg/libogg-1.3.4-r1.ebuild:30]: don't change this, don't want to call `make install` in the `_all` phase

File: media-libs/opus/opus-1.3.1-r1.ebuild [hunk 6]
```diff
@@ -26,9 +26,11 @@ multilib_src_configure() {
 		$(use_enable doc)
 		$(use_enable static-libs static)
 	)
+
 	for i in ${INTRINSIC_FLAGS} ; do
```
Comment (SoapGentoo) [inline media-libs/opus/opus-1.3.1-r1.ebuild:30]: `local i`

File: media-libs/opus/opus-1.3.1-r2.ebuild [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+		$(use_enable static-libs static)
+	)
+
+	local i
+	# We want to disable intrinsics if no flags are enabled
+	# (This is a fun Bash construct to do that!)
+	# bug #752069
+	for i in "${INTRINSIC_FLAGS}" ; do
+		use ${i} && myeconfargs+=( --enable-intrinsics ) && break
+	done ||  myeconfargs+=( --disable-intrinsics )
```
Comment (SoapGentoo) [inline media-libs/opus/opus-1.3.1-r2.ebuild:42]:
```suggestion
	done || myeconfargs+=( --disable-intrinsics )
```

File: media-libs/opus/opus-1.3.1-r2.ebuild [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+	if is-flagq -ffast-math || is-flagq -Ofast; then
+		myeconfargs+=( --enable-float-approx )
+	fi
+
+	ECONF_SOURCE="${S}" econf "${myeconfargs[@]}"
+}
+
+multilib_src_install_all() {
+	default
```
Comment (SoapGentoo) [inline media-libs/opus/opus-1.3.1-r2.ebuild:52]:
```suggestion
	einstalldocs
```

**Pull request closed without merging**